### DISCUSSION TITLE
Windows 11 version 22H2 Notice

### DISCUSSION THREAD

**Discussion by absolutedomie** [AveYo/MediaCreationTool.bat#193] [Ideas]
Hey there, since Windows 11 version 22H2 is out, Can You Add that to The Script?

**Comment by AveYo** [comment]
I have it almost ready, will update probably tomorrow!

**Reply by ghost** [reply]
Hi, I tested this amazing script, now when trying to update the earlier Win 11 installation made to 22H2 from the ISO file from today (Windows Update didn't show any 22H2, I was curious, so I went on with a downloaded ISO file), there was no option to keep the already installed apps etc. I went on anyway just to have a look at the new 22H2, and as it said, the previously installed things were gone.

**Comment by GoodMood1974** [comment]
Hi, the current script does not support Windows 11 22H2. (i tried to upgrade from 21H2).
Are you going to release a new script soon?
Thank you!

**Reply by xversion1** [reply]
> Hi, the current script does not support Windows 11 22H2. (i tried to upgrade from 21H2). Are you going to release a new script soon? Thank you!

I used the script to get my 21H2 update too. It didn't work. I thought something was wrong with my PC because some recent articles still use this script to guide people to get the new update. It turns out we just need to wait for a new fix.

**Comment by AveYo** [comment]
Postponed revision as I went a bit down the rabbit-hole trying to fix flaws in MCT, but the end result is gonna be great!

**Reply by zshadow1803** [reply]
Hello Ave,

I hope you find solutions to these problems.
We are eagerly awaiting the new version.

Cheers.

**Comment by dajoey** [comment]
any eta?
